voter/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 order: 1
-description: Create a simple blockchain poll application, add types and messages, and design the front-end app.
+description: Create a blockchain poll application, add types and messages, and design the front-end app.
 ---
 
 # Polling app
 
-This tutorial creates a simple blockchain poll application.
+This tutorial creates a blockchain poll application.
```

Comment:
:praise:  I see you are embracing the Digital Ocean philosophy of not using "simple" good job!